docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://azconfig.io/schemas/FeatureManagement/v1.0.0/FeatureManagement.json",
+    "type": "object",
+    "title": "An Azure App Configuration Feature Management Configuration",
```

Comment:
I would say this is Microsoft Feature Management schema.

The schema that describes individual flag is the `Microsoft Feature Flag Schema`.

Reply:
I like the idea naming this schema as `Microsoft Feature Management`.
@zhenlan 

As discussed in the email thread, snake_casing will be used:
```json
"feature_management": {
  "feature_flags": [
     {
       // Microsoft feature flag schema
     }
  ]
}

```